adobe_vipm/airtable/models.py

```diff
@@ -264,88 +277,88 @@ class Meta:
     return Offer
 
 
-def get_offer_ids_by_membership_id(product_id, membership_id):
+def get_offer_ids_by_membership_id(product_id: str, membership_id: str) -> list[str]:
     """
     Returns a list of SKUs associated with a given membership_id.
 
     Args:
-        product_id (str): The ID of the product to which the membership refers to.
-        membership_id (str): The membership ID used to retrieve the list of SKUs.
+        product_id: The ID of the product to which the membership refers to.
+        membership_id: The membership ID used to retrieve the list of SKUs.
 
     Returns:
         list: List of SKUs that belong to the given membership.
     """
-    Offer = get_offer_model(AirTableBaseInfo.for_migrations(product_id))
-    return [
-        offer.offer_id
-        for offer in Offer.all(formula=EQUAL(FIELD("membership_id"), STR_VALUE(membership_id)))
-    ]
+    offer_model = get_offer_model(AirTableBaseInfo.for_migrations(product_id))
+    formula = EQUAL(FIELD("membership_id"), STR_VALUE(membership_id))
+
+    return [offer.offer_id for offer in offer_model.all(formula=formula)]
 
 
-def create_offers(product_id, offers):
+def create_offers(product_id: str, offers: list):
     """
     Creates a list of Offer objects in batch.
 
     Args:
-        product_id (str): The ID of the product used to determine the AirTable base.
-        offers (list): List of Offer objects to create.
+        product_id: The ID of the product used to determine the AirTable base.
+        offers: List of Offer objects to create.
     """
-    Offer = get_offer_model(AirTableBaseInfo.for_migrations(product_id))
-    Offer.batch_save([Offer(**offer) for offer in offers])
+    offer_model = get_offer_model(AirTableBaseInfo.for_migrations(product_id))
+    offer_model.batch_save([offer_model(**offer) for offer in offers])
 
 
-def get_transfers_to_process(product_id):
+def get_transfers_to_process(product_id: str):
     """
     Get a list of transfers that must be submitted to Adobe.
 
     Args:
-        product_id (str): The ID of the product used to determine the AirTable base.
+        product_id: The ID of the product used to determine the AirTable base.
 
     Returns:
         list: List of Transfer objects.
     """
-    Transfer = get_transfer_model(AirTableBaseInfo.for_migrations(product_id))
-    return Transfer.all(
+    transfer_model = get_transfer_model(AirTableBaseInfo.for_migrations(product_id))
+    return transfer_model.all(
         formula=OR(
             EQUAL(FIELD("status"), STR_VALUE(STATUS_INIT)),
             EQUAL(FIELD("status"), STR_VALUE(STATUS_RESCHEDULED)),
         ),
     )
 
 
-def get_transfers_to_check(product_id):
+def get_transfers_to_check(product_id: str):
     """
```

Comment:
Looks like we also forgot the return typing in the methods below

```suggestion
def get_transfers_to_check(product_id: str) -> list:
    """
```